integration-tests/build.gradle

```diff
@@ -9,11 +9,8 @@ repositories {
     maven { url "https://splunk.jfrog.io/splunk/ext-releases-local" }
 }
 ext {
-    besuPluginVersion = '25.2.1'
-    besuInternalVersion = '25.2.1'
-    besuInternalCryptoVersion = '23.1.3'
-    besuCryptoDepVersion = '1.1.2'
-    besuBlsVersion = '1.0.0'
+    besuDepVersion = '25.10.0'
+    besuNativeDepVersion = '1.4.0'
```

Comment:
I hope this simplification hasn't side effects, to better understand, is there a reason to keep some of the dependencies on older version: `besuInternalCryptoVersion = '23.1.3'` and `besuBlsVersion = '1.0.0'`?